src/lib/ng-bootstrap-form-validation.module.ts

```diff
@@ -21,6 +21,14 @@ export class NgBootstrapFormValidationModule {
   static forRoot(
     customErrorMessages: ErrorMessage[] = []
   ): ModuleWithProviders {
+    if (customErrorMessages.length) {
+      console.warn(
+        "Deprecation warning: Passing 'customErrorMessages' to " +
+          "the 'forRoot' method is deprecated and will be removed in a future " +
+          "release. Please use the 'CUSTOM_ERROR_MESSAGES' multi-provider."
```

Comment:
Would be nice if we could link to an example of how to do this for developer convenience.

Reply:
This doesn't necessarily need to hold up this PR though.

Reply:
I updated the readme docs on using custom error messages and put a link to that in the deprecation message.